docs/service_spec.md

```diff
@@ -124,6 +124,26 @@ and will send a `?filter=name` query parameter along with streaming/polling requ
 For tests that involve filtering, the test harness will set the `filter` property of the `streaming` or `polling` configuration
 object. The property will either be omitted if no filter is requested, or a non-empty string if requested.
 
+### Capability `"evaluation-hooks"`
+
+This means that the SDK has support for hooks and has the ability to register evaluation hooks.
+
+For a test service to support hooks testing it must support a `test-hook`. The configuration will specify registering one or more `test-hooks`.
+
+A test hook must:
+  - Implement the SDK hook interface.
+  - Whenever an evaluation stage is called post information about that call to the `callbackUrl` of the hook.
+    - The payload is an object with the following properties:
+      * `evaluationHookContext` (object, optional): If an evaluation stage was executed, then this should be the associated context.
+        * `flagKey` (string, required): The key of the flag being evaluated.
+        * `context` (object, required): The evaluation context associated with the evaluation.
+        * `defaultValue` (any): The default value for the evaluation.
+        * `method` (string, required): The name of the evaluation emthod that was called.
+      * `evaluationHookData` (object, optional): The EvaluationHookData passed to the stage during execution.
+      * `evaluationDetail` (object, optional): The details of the evaluation if executing an `afterEvaluation` stage.
+      * `stage` (string, optional): If executing a stage, for example `beforeEvaluation`, this should be the stage.
+  - Return data from the stages as specified via the `data` configuration. For instance the return value from the `beforeEvaluation` hook should be `data['beforeEvaluation']` merged with the input data for the stage.
```

Comment:
```suggestion
    - The payload is an object with the following properties:
      * `evaluationSeriesContext` (object, optional): If an evaluation stage was executed, then this should be the associated context.
        * `flagKey` (string, required): The key of the flag being evaluated.
        * `context` (object, required): The evaluation context associated with the evaluation.
        * `defaultValue` (any): The default value for the evaluation.
        * `method` (string, required): The name of the evaluation emthod that was called.
      * `evaluationSeriesData` (object, optional): The EvaluationSeriesData passed to the stage during execution.
      * `evaluationDetail` (object, optional): The details of the evaluation if executing an `afterEvaluation` stage.
      * `stage` (string, optional): If executing a stage, for example `beforeEvaluation`, this should be the stage.
  - Return data from the stages as specified via the `data` configuration. For instance the return value from the `beforeEvaluation` hook should be `data['beforeEvaluation']` merged with the input data for the stage.
```

To be consistent with naming updates. Although, now that I am reading it here, perhaps Stage was the right word instead of series - the context/data applies to all elements of the series, but is seen (by the customer) in a single stage.